### PR TITLE
Update intellij config

### PR DESCRIPTION
**Pull request by f1sh1918**
There were some adjustments in the intejji config missing or incorrect:

- add font-glypth path for map styles
- adjust run config for administration test
- remove unneeded config file

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -10,5 +10,6 @@
   <component name="VcsDirectoryMappings">
     <mapping directory="" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/backend/ehrenamtskarte-maplibre-style" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/backend/ehrenamtskarte-maplibre-style/font-glyphs" vcs="Git" />
```
Comment (sarahsporck) [inline .idea/vcs.xml:13]: You should add to the docs then that this submodule also needs to be initialized (as I did not do this yet ^.^)
Running `git submodule update --init --recursive` in the root repo `entitlementcard` will recursively initialize all submodules.

Comment (f1sh1918) [reply .idea/vcs.xml:13]: ok so is this even needed then? Should be cloned auto?
```
8. Clone the submodule inside `backend/ehrenamtskarte-maplibre-style`
```

**Review by sarahsporck — APPROVED** [review]

File: .idea/vcs.xml (hunk 1 above)
Comment (f1sh1918) [reply .idea/vcs.xml:13]: you still didnt't tell if you checked if step 8 (clone submodule) is needed? I would like to update the docs

Comment (sarahsporck) [reply .idea/vcs.xml:13]: It seems I had to update git, so that the submodules would be properly initialized on clone.

**Pull request merged**